lib/cc/engine/markdownlint.rb

```diff
@@ -7,33 +7,36 @@ module Engine
     class Markdownlint
       CONFIG_FILE = "./.mdlrc".freeze
       EXTENSIONS = %w[.markdown .md].freeze
+      UnexpectedMessageFormat = Class.new(StandardError)
 
-      def initialize(root, engine_config, io)
+      def initialize(root, engine_config, io, err_io)
         @root = root
         @engine_config = engine_config
         @io = io
+        @err_io = err_io
         @contents = {}
       end
 
       def run
         return if include_paths.length == 0
 
-        pid, _, out, err = POSIX::Spawn.popen4("mdl", *mdl_options, *include_paths)
+        child = POSIX::Spawn::Child.new("mdl", *mdl_options, *include_paths)
```

Comment:
I think the best answer would be a PR on mivok/markdownlint to add an option to exit zero even if issues are found. I would check around on some other tools, I think some call it `--quiet`, but I'm not a fan of that; maybe `--supress-error`?